master/pkg/stream/stream.go

```diff
@@ -109,17 +131,18 @@ func (p *Publisher[T]) Broadcast(events []Event[T]) {
 
 	// check each event against each subscription
 	for _, sub := range p.Subscriptions {
-		func(){
+		func() {
 			for _, ev := range events {
 				var msg *websocket.PreparedMessage
-				if ev.After != nil && sub.filter(*ev.After) {
-					// update, insert, or fallin: send the record to the client.
+				switch {
+				// update, insert, or fallin: send the record to the client.
```

Comment:
same comment about how things are worded vs where they appear relative to the execution of the conditional.